guides/doc-Release_notes/topics/foreman-3.3.0.adoc

```diff
@@ -0,0 +1,266 @@
+= Foreman 3.3.0
+
+A full list of changes is available via https://projects.theforeman.org/issues?set_filter=1&sort=id%3Adesc&status_id=closed&f%5B%5D=cf_12&op%5Bcf_12%5D=%3D&v%5Bcf_12%5D%5B%5D=1564[Redmine]
+
+== Foreman
+
+* Support EC (and other non-RSA) keys in Foreman core - https://projects.theforeman.org/issues/34843[#34843]
+* Remove the "Share your feedback"  link from the new host details page - https://projects.theforeman.org/issues/34841[#34841]
+* Add a reports tab to the hosts details page - https://projects.theforeman.org/issues/34782[#34782]
+* Add developer documentation for pagelets - https://projects.theforeman.org/issues/34544[#34544]
+* Include the host comments to a host list page - https://projects.theforeman.org/issues/34516[#34516]
+* Host list page should show the owner of the host - https://projects.theforeman.org/issues/34515[#34515]
+
+=== Settings
+
+* Fix the SettingValueException error message - https://projects.theforeman.org/issues/34921[#34921]
+* Settings - Issues with search - https://projects.theforeman.org/issues/34866[#34866]
+* "Sync Connect Timeout" settings takes invalid value and shows update successful but doesnt reflect the change for invalid valuest reflect the change for invalid values - https://projects.theforeman.org/issues/34838[#34838]
```

Comment:
"Sync Connect Timeout" settings takes" should be "Sync Connect Timeout" setting takes".